requirements.txt

```
@@ -1,4 +1,4 @@
-dwave-ocean-sdk>=3.0.0
+dwave-ocean-sdk>=3.2.1
```

Labels are added in 3.3:
```suggestion
dwave-ocean-sdk>=3.3.0
```